backend/logic/order/index.js

```diff
@@ -194,6 +194,30 @@ async function processNewOrder({
   const order = await Order.create(orderObj)
   log.info(`Saved order ${order.fqId} to DB.`)
 
+  if (shopConfig.inventory) {
+    const cartItems = get(data, 'items', [])
+    const dbProducts = await Product.findOne({
```

Comment:
What if any of the item in the cart is not found in the DB? shouldn't this be an error?